lib/primitives/mtx.js

```diff
@@ -1413,6 +1413,27 @@ class MTX extends TX {
 /**
  * Coin Selector
  * @alias module:primitives.CoinSelector
+ * @property {TX} tx - clone of the original mtx.
+ * @property {CoinView} view - reference to the original view.
+ * @property {Coin[]} coins - List of available coins.
+ * @property {Amount} outputValue - Total output value.
+ * @property {Coin[]} chosen - List of chosen coins to add.
+ * @property {Coin[]} viewCoins - List of coins to only apply to coinview.
+ * @property {Address} changeAddress - change address.
+ * @property {Amount} change - change value.
+ * @property {Amount} fee - Value of fee.
+ * @property {String} selection - selection type.
+ * @property {Boolean} subtractFee - whether to subtract fee from the output.
+ * @property {Number} subtractIndex - index of the output to subtract fee from.
+ * @property {Number} height - height of the chain. (to check spendability)
+ * @property {Number} depth - depth or confirmations for the coin.
+ * @property {Number} hardFee - fixed fee.
+ * @property {Number} rate - Rate of dollarydoo per kB.
+ * @property {Number} maxFee - maximum fee we are willing to pay.
+ * @property {Boolean} round - round to the nearest kilobyte.
+ * @property {Number} coinbaseMaturity - when do CBs become spendable.
+ * @property {Function} estimate - Input script size estimator.
+ * @property {Object[]} inputs - preferred inputs.
```

Comment:
There's more confusion here because we have `inputs` which you describe as "preferred inputs".... but we also have object property `preferredInputs` 😬

Reply:
IIUC, `inputs` is actually the original tx existing inputs and `preferredInputs` are passed in as part of the options object